frontend/lib/src/components/elements/ImageList/ImageList.tsx

```diff
@@ -95,7 +110,7 @@ export function ImageList({
       data-testid="stImage"
       style={{ width }}
     >
-      {element.imgs.map((iimage: IImage, idx: number): ReactElement => {
+      {element.imgs.map((iimage, idx): ReactElement => {
```

Comment:
intentional removal of types?

Reply:
Yes! These are unnecessary since they are already properly inferred by the TypeScript compiler. Keeping them here gives us no additional type safety, and only increases the dev burden if the underlying definitions were to change.

Reply:
💯% agree on that. what's your opinion on our explicit return type requirement?

Reply:
Definitely a conversation for another forum, but it's unnecessary in a majority of cases for similar reasons above 🙂

Reply:
👍 cool cool, I agree with that too. I'll create an item in the notion doc regarding typescript options :D